indico/modules/rb_new/controllers/backend/admin.py

```diff
@@ -69,6 +70,7 @@ class SettingsSchema(mm.Schema):
     end_notification_weekly = fields.Int(validate=[validate.Range(min=1, max=30)])
     end_notification_monthly = fields.Int(validate=[validate.Range(min=1, max=30)])
     excluded_categories = ModelList(Category)
+    grace_period = fields.Int(missing=None, validate=[validate.Range(max=24)])
```

Comment:
- `missing=None` is wrong here - you now always set the grace period to null when saving something else and not touching this field at all!
- missing `min` in the range
- probably needs `allow_none=True`